src/main/java/org/terracotta/offheapstore/paging/UpfrontAllocatingPageSource.java

```diff
@@ -480,17 +480,17 @@ public synchronized Runnable removeAllocationThreshold(ThresholdDirection direct
             allocatorLog.printf("%d,%d,%d,%d,%d,%d,%d,%d%n", System.nanoTime() - start, blockDuration, b.capacity(), allocated, PhysicalMemory.freePhysicalMemory(), PhysicalMemory.totalSwapSpace(), PhysicalMemory.freeSwapSpace(), PhysicalMemory.ourCommittedVirtualMemory());
           }
           if (allocated > nextProgressLogAt) {
-            LOGGER.info("Allocation {}% complete", (100 * allocated) / max);
+            LOGGER.debug("Allocation {}% complete", (100 * allocated) / max);
```

Comment:
This one is a bit of a problem: on very large offheaps like TB ones, this is the only indication that progress is being made. This is handy to have as the allocation can take minutes.
Maybe we should only display it if the allocation takes over X seconds?